test/util/load_sku_packs.py

```diff
@@ -2,30 +2,34 @@
 Copyright 2016, EMC, Inc.
 
 Author(s):
-George Paulos
 
 This script load SKU packs from sources specified in install_default.json
 '''
 
-
 import os
 import sys
 import subprocess
+
 # set path to common libraries
 sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test/common")
 import fit_common
 
-class load_sku_packs(fit_common.unittest.TestCase):
-    def test01_preload_sku_packs(self):
-        print "**** Processing SKU Packs"
-        # Load SKU packs from GutHub
+from nose.plugins.attrib import attr
+@attr(all=True)
+
+class fit_template(fit_common.unittest.TestCase):
+
+    def test01_download_sku_packs(self):
+        # Download SKU packs from GitHub
         subprocess.call("rm -rf temp.sku; rm -rf on-skupack", shell=True)
         os.mkdir("on-skupack")
```

Comment:
Should check the return of this call to make sure it succeeded since much of the rest of the tests depends on this.